s3commander.js

```diff
@@ -641,8 +641,14 @@ b64pad = "=";
       // file control
       return (
         <div className={this.props.style.entry}>
-          <span className="glyphicon glyphicon-file"></span>
-          <a onClick={this.onDownload}>{file.name}</a>
+          <span
+            onClick={this.onDownload}
+            className="s3icon glyphicon glyphicon-file">
```

Comment:
Should the s3icon class be on the <a> as well?  Or does the later change (L819) make it so that links get this property?